mqtt/src/test/java/akka/stream/alpakka/mqtt/javadsl/MqttSourceTest.java

```diff
@@ -45,12 +47,19 @@
 
   @BeforeClass
   public static void setup() throws Exception {
+    serverPort = AvailablePort$.MODULE$.nextPort();
+    serverWebSocketPort = AvailablePort$.MODULE$.nextPort();
```

Comment:
I am not familiar with Java code calling Scala code I will take a look if `AvailablePort.nextPort();` will work, thanks